src/models/README.md

```diff
@@ -10,6 +10,7 @@ This documentation includes the following System76 models:
 - [Darter Pro (darp7)](darp7/README.md)
 - [Galago Pro (galp5)](galp5/README.md)
 - [Gazelle (gaze16)](gaze16/README.md)
+- [Kudu (kudu6)](models/kudu6/README.md)
```

Comment:
```suggestion
+ [Kudu (kudu6)](kudu6/README.md)
```

Reply:
Fixed with f8c20f5, thank you for catching that.